### DISCUSSION TITLE
@formily/antd/esm/space/index.js 写法导致 编译报错。

### DISCUSSION THREAD

**Discussion by dahui4dev** [alibaba/formily#1608] [Q&A 2.x]
本地 cra 创建的工程，使用 `@formily/antd` 包里的 `Space` 组件之后，编译报错：

本地工程依赖版本：

```
    "@formily/antd": "2.0.0-beta.66",
    "@formily/core": "2.0.0-beta.66",
    "@formily/react": "2.0.0-beta.66",
    "@formily/reactive": "2.0.0-beta.66",
```

文件路径：`@formily/antd/esm/space/index.js`

报错如下：

```
Starting the development server...
Failed to compile.

./node_modules/@formily/antd/esm/space/index.js 3:9
Module parse failed: Export 'Space' is not defined (3:9)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| import "antd/lib/space/style";
| import _Space from "antd/lib/space";
> export { Space };
| export default _Space;

```

搜索了一下和这个问题应该是同一个问题。https://github.com/alibaba/formily/discussions/1301

改成下图这样就好了，不知是不是 bug。
![image](https://user-images.githubusercontent.com/6160798/122701005-eb47de00-d27e-11eb-9543-27028574366a.png)

**Comment by dahui4dev** [comment]
编译问题，不知道怎么搞 codesandbox

**Reply by liuweiGL** [reply]
给个 github 复现仓库也可以的。

**Reply by xiaxiedaima** [reply]
> 编译问题，不知道怎么搞 codesandbox

你也是用的craco-antd加载less吗?我换成craco-less好像没报错了

**Reply by atzcl** [reply]
```ts
export { xxx }
```
 的用法是没问题的，这个更应该是 `babel` 的问题

**Reply by atzcl** [reply]
可以尝试下在 `babel` 的配置里面，对 `@babel/plugin-transform-runtime` 的配置加多一个 `"useESModules": true`，或者更新 `babel` 到最新版本
